build_debian.sh

```diff
@@ -208,6 +208,23 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get update
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install docker-ce=${DOCKER_VERSION}
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y remove software-properties-common gnupg2
 
+if [ "$INSTALL_KUBERNETES" == "y" ]
+then
+    KUBERNETES_VERSION="${KUBERNETES_VERSION:-1.18.0-00}"
+    ## Install Kubernetes
+    echo '[INFO] Install kubernetes'
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT curl -o /tmp/k8s.gpg -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-key add /tmp/k8s.gpg
+    sudo LANG=C chroot $FILESYSTEM_ROOT rm /tmp/k8s.gpg
+    sudo cp files/image_config/kubernetes/kubernetes.list $FILESYSTEM_ROOT/etc/apt/sources.list.d/
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get update
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -qy install kubeadm=${KUBERNETES_VERSION}
```

Comment:
>q [](start = 49, length = 1)

Keep them verbose

Reply:
Sorry. I don't get this comment. Can you  please elaborate?

Reply:
```suggeston
sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -q install kubeadm=${KUBERNETES_VERSION}
```

Reply:
```
sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -q install kubeadm=${KUBERNETES_VERSION}
```

---
In reply to: [404416407](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r404416407) [](ancestors = 404416407)

Reply:
I still don't get it, as the original line and your comments appear identical. BTW, there is a minor change in this line in updated code.

Reply:
Drop -q -- so verbose o/p will be there